[ADD] Time Off: Moving TO request to new doc (#9431)
odoo/documentation: content/applications/hr/time_off/request_time_off.rst

================
Request time off
================

Once time off has been allocated to an employee, a request to use it can be submitted.

> Time off can be requested in one of two ways: either from the main *Time Off* application dashboard

Reviewer: ```suggestion
Time off can be requested in one of two ways: either from the main *Time Off* application :guilabel:`Dashboard`
```

(:menuselection:`Time Off app --> My Time --> Dashboard`), or from the :guilabel:`My Time Off`
dashboard view (:menuselection:`Time Off app --> My Time --> My Time Off`).

To create a new request for time off, click the :guilabel:`New` button on either the main *Time Off*

> :guilabel:`Dashboard` or the :guilabel:`My Time Off` list view.

Reviewer: ```suggestion
:guilabel:`Dashboard` or the :guilabel:`My Time Off` dashboard, in the default list view.
```

.. note::
   Both :guilabel:`New` buttons allow the user to request time off, but when requested from the

>    :guilabel:`Dashboard`, a :guilabel:`New Time Off` request form appears in a modal. When requested

Reviewer: ```suggestion
   :guilabel:`Dashboard`, a :guilabel:`New Time Off` request form appears in a pop-up window. When requested
```

   from the :guilabel:`My Time Off` list view, the screen navigates to a new time off request page,
   instead.

Enter the following information on the :guilabel:`New Time Off` request form:

- :guilabel:`Time Off Type`: select the type of time off being requested from the drop-down menu.
- :guilabel:`Dates`: enter the dates that the time off falls under. There are two fields to
  populate, the start and end dates. Click on either date field and a popover calendar appears.

  Click on the start date, then click on the end date. The selected start and end dates are circled,
  and the dates between them are highlighted (if applicable).

  If the time off requested is for a single day, click on the start date, then click the same date
  again for the end date.

>   When the correct dates are selected/highlighted, click the :guilabel:`Apply` button.

Reviewer 2: ```suggestion
  When the correct dates are selected, click the :guilabel:`Apply` button.
```

  The selected dates now populate the two portions of the :guilabel:`Dates` field.

  If the selected :guilabel:`Time Off Type` is configured to have the time off taken in hours, the
  following two fields also appear:

>   - :guilabel:`Half Day`: if the time off request is for a half day, check this box. When this is

Reviewer: ```suggestion
  - :guilabel:`Half Day`: if the time off request is for a half day, tick this checkbox. When this is
```

    selected, the second date field disappears, and is replaced with a drop-down menu. From that
    drop-down menu, select either :guilabel:`Morning` or :guilabel:`Afternoon` to indicate which
    half of the day is being requested.

>   - :guilabel:`Custom Hours`: if the time off requested is not a whole or half day, check this box.

Reviewer: ```suggestion
  - :guilabel:`Custom Hours`: if the time off requested is not a whole or half day, tick this checkbox.
```

>     A :guilabel:`From` and :guilabel:`To` field appears beneath this option, if selected. Using the

Reviewer 2: ```suggestion
    If selected, a :guilabel:`From` and :guilabel:`To` field appears beneath this option. Using the
```

    drop-down menu, select the start and end time for the time off request.

- :guilabel:`Duration`: this field updates automatically once the :guilabel:`Date` section is
  completed. If the :guilabel:`Date` section is modified, this section automatically updates to
  reflect the total time off requested. This field is in either hours or days, depending on how the
  selected :guilabel:`Time Off Type` is configured.
- :guilabel:`Description`: enter a description for the time off request. This should include any

>   details that managers and approvers may need in order to approve the request.

Reviewer 2: ```suggestion
  details that managers and approvers may need to approve the request.
```

- :guilabel:`Supporting Document`: this field only appears if the :guilabel:`Time Off Type` selected

>   allows for the attachments of documents. Click the :guilabel:`Attach File` button, and a file

Reviewer 2: ```suggestion
  allows for the attachment of documents. Click the :guilabel:`Attach File` button, and a file
```

  explorer window appears.

>   Navigate to the desired files to attach, select them, then click the :guilabel:`Open` button. The

Reviewer: ```suggestion
  Navigate to the desired files that should be attached, select them, then click the :guilabel:`Open` button. The
```

  files then appear on the time off request form. Multiple documents can be attached, if necessary.

If the request was created from the :guilabel:`Dashboard`, click the :guilabel:`Save & Close` button

> to save the information and submit the request.

Reviewer: ```suggestion
to save the information, and submit the request.
```

If the form was completed from the :guilabel:`My Time Off` list view, the information is
automatically saved as it is entered. However, the form can be saved manually at any time by

> clicking the :icon:`fa-cloud-upload` :guilabel:`(save manually)` icon.

Reviewer: I'm a bit torn on this... not sure if it should be :guilabel:`Save manually' or :guilabel:`(cloud upload)`

Might want to check in with SALI about this one to see what he thinks is best...

Because, currently, if that icon fails to load, the only thing the reader will see is the words '(save manually)` icon, which doesn't provide a lot of visual context... but I *do* think there's a benefit to educating the reader about what that icon actually *does* represent...

So, yeah, all that to say...check in with SALI about this one... 👍

Reviewer 3: Hi @larm-odoo and @ksc-odoo, I'd prefer to use the icon's name in the label :) because it effectively describes the icon and can be referred to consistently by multiple writers. For example, if we use the icon/button's action or purpose in the :guilabel:, it could be inconsistent across docs. Some may say "(Manual save)" or "(Save)".
```suggestion
clicking the :icon:`fa-cloud-upload` :guilabel:`(cloud upload)` icon.
```
